src/test/java/seedu/address/logic/parser/FindLeadStatusCommandParserTest.java

```diff
@@ -0,0 +1,35 @@
+package seedu.address.logic.parser;
+
+import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseFailure;
+import static seedu.address.logic.parser.CommandParserTestUtil.assertParseSuccess;
+
+import java.util.Arrays;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.FindLeadStatusCommand;
+import seedu.address.model.person.FindContainsStatusPredicate;
+
+public class FindLeadStatusCommandParserTest {
+
+    private FindLeadStatusCommandParser parser = new FindLeadStatusCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ",
+                String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindLeadStatusCommand.MESSAGE_USAGE));
+    }
+
+    @Test
+    public void parse_validArgs_returnsFindCommand() {
```

Comment:
Might be good to rename `parse_validArgs_returnsFindCommand()` to `parse_validArgs_returnsFindLeadStatusCommand()` to differentiate between test cases